.github/workflows/api_refs.yaml

```diff
@@ -0,0 +1,54 @@
+name: 'Build API Refs'
+
+on:
+    workflow_dispatch:
+        inputs:
+            version:
+                description: 'Released version'
+                required: true
+                type: string
+
+jobs:
+    open_php_api_ref_pr:
+        name: "PHP & REST API References' PR"
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Set version and branches
+              id: version_and_branches
+              run: |
+                  version="${{ inputs.version }}"
+                  base_branch="$(echo $version | sed -s /v(.*\..*)\..*/\1/)"
+                  work_branch="api_refs_$version"
+                  echo "version=$version" >> "$GITHUB_OUTPUT"
+                  echo "base_branch=$base_branch" >> "$GITHUB_OUTPUT"
+                  echo "work_branch=$work_branch" >> "$GITHUB_OUTPUT"
+
+            - name: Checkout documentation
+              uses: actions/checkout@v4
+
+            - name: Build API Refs
+              # TODO: On 5.0, update tool to handle both PHP & REST APIs
```

Comment:
To do this, few lines can be added to [the 5.0 version of phpdoc.sh](https://github.com/ibexa/documentation-developer/pull/2710/files):

```php
$PHP_BINARY bin/console ibexa:openapi --yaml > openapi.yaml
redocly build-docs openapi.yaml --output $(realpath $OUTPUT_DIR/../../rest_api/rest_api_reference/rest_api_reference.html)
```

Anyway, the tool should be renamed and reworked to reflect its new task of building both references.